### PR TITLE
fix typo

### PR DESCRIPTION
**Pull request by x1957**
fix typo in comment.

### CODE REVIEW

**Comment by hicqu** [comment]
@x1957 Would you mind to fix the CI?

**Comment by x1957** [comment]
> @x1957 Would you mind to fix the CI?

What's wrong with CI?

**Comment by ice1000** [comment]
@x1957 just merge the current master branch can solve this

**Comment by x1957** [comment]
> @x1957 just merge the current master branch can solve this

Thanks.

**Comment by x1957** [comment]
@nrc @hicqu 
Fix CI & fix more "commited" typo.

File: src/progress/progress_set.rs [hunk 1]
```diff
@@ -155,7 +155,7 @@ pub struct ProgressSet {
     #[get = "pub"]
     next_configuration: Option<Configuration>,
     configuration_capacity: (usize, usize),
-    // A preallocated buffer for sorting in the minimally_commited_index function.
+    // A preallocated buffer for sorting in the minimally_committed_index function.
```
Comment (ice1000) [inline src/progress/progress_set.rs]: Did you rename the actual function as well?

Comment (x1957) [reply src/progress/progress_set.rs]: I can't find this function, just found ***maximal_committed_index***

Comment (ice1000) [reply src/progress/progress_set.rs]: Then I'll state this as a bigger problem -- it's probably a non-trivial typo.

Git blame shows that this commit is https://github.com/pingcap/raft-rs/tree/88e3697c985eb55c599add7b5cd24a40fd05ebc3, where at that time I can't find such function as well :(

/cc @Hoverbear

Comment (ice1000) [reply src/progress/progress_set.rs]: <pre><font color="#3465A4">~/git-repos/raft-rs</font><font color="#2E3436">|(88e3697...)</font> <font color="#C4A000">λ&gt;</font> grep -rnE &quot;minimally_commited_index&quot; src
<font color="#75507B">src/progress.rs</font><font color="#06989A">:</font><font color="#4E9A06">82</font><font color="#06989A">:</font>    // A preallocated buffer for sorting in the <font color="#EF2929"><b>minimally_commited_index</b></font> function.
</pre>

Comment (hicqu) [reply src/progress/progress_set.rs]: It's a stale comment. Just rename it to `maximal_committed_index` is OK.

**Comment by ice1000** [comment]
@hicqu @nrc PTAL

File: src/progress/progress_set.rs (hunk 1 above)
Comment (Hoverbear) [reply src/progress/progress_set.rs]: Yup that's my bad, sorry.

**Comment by ice1000** [comment]
Travis CI crashed nightly only. Merge?

**Comment by Hoverbear** [comment]
Sigh, I think we need to disable windows nightly until Travis fixes it.

**Pull request merged**